fixieai/cli/agent/commands.py

```diff
@@ -241,7 +245,21 @@ def serve(ctx, path, host, port, tunnel):
             console.print(f"{host}:{port} can be reached at {config.deployment_url}")
 
         agent_api = _ensure_agent_updated(ctx.obj.client, config)
-        agent_impl.serve(agent_api.agent_id, host, port)
+
+        if reload:
+            os.environ["FIXIE_REFRESH_AGENT_ID"] = agent_api.agent_id
+            uvicorn.run(
+                config.entry_point + ".app",
+                host=host,
+                port=port,
+                factory=True,
+                reload=True,
+                reload_dirs=[os.path.dirname(path)],
+                app_dir=os.path.dirname(path),
+            )
+        else:
+            _, agent_impl = loader.load_agent_from_path(path)
+            agent_impl.serve(agent_api.agent_id, host, port)
```

Comment:
Maybe get rid of `load_agent_from_path`(/`serve`?) altogether and just use `uvicorn` directly in both deploy/serve scenarios.

Reply:
Thinking about this more, I think this is worth doing eventually (probably after hackathon), but to minimize risk of incompatible SDK versions during testing or otherwise I'll wait for this change to roll out first.